package-system/OpenSSL/build_package_image.py

```diff
@@ -29,8 +31,9 @@ def main():
 
     packageSystemDir = Path(__file__).resolve().parents[1]
     opensslPackageSourceDir = packageSystemDir / 'OpenSSL'
-    outputDir = packageSystemDir / f'OpenSSL-{args.platformName}'
-    opensslPatch = opensslPackageSourceDir / 'set_openssl_port_to_1_1_1_m.patch'
+    outputDir = opensslPackageSourceDir / 'temp' / f'OpenSSL-{args.platformName}'
+
+    opensslPatch = opensslPackageSourceDir / 'set_openssl_port_to_1_1_1_x.patch'
```

Comment:
Note that this was named _x intentionally since we could then reuse it for o, p, q, etc, without renaming a file and make diffs cleaner in the future.